src/main/drivers/sonar_hcsr04.c

```diff
@@ -134,30 +138,33 @@ void hcsr04_init(const sonarHardware_t *initialSonarHardware, sonarRange_t *sona
     NVIC_InitStructure.NVIC_IRQChannelCmd = ENABLE;
     NVIC_Init(&NVIC_InitStructure);
 
-    lastMeasurementAt = millis() - 60; // force 1st measurement in hcsr04_get_distance()
-#else
-    lastMeasurementAt = 0; // to avoid "unused" compiler warning
 #endif
 }
 
-// measurement reading is done asynchronously, using interrupt
+#ifdef UNIT_TEST
+void digitalHi(void *p, uint16_t i) {UNUSED(p);UNUSED(i);}
```

Comment:
These are deliberate, to facilitate the writing of test code. The sonar code is compiled into 3 of the unit tests: `flight_imu_unittest`,  `altitude_hold_unittest`, and `sonar_unittest`. Having these stubs here avoids the need for multiple stubs in the test code.

This can be changed if it is not considered good practice.